src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -270,18 +270,18 @@ export const useSendThorTx = ({
     if (!accountId) return
     if (!transactionType) return
     if (!estimateFeesArgs) return
-    if (accountNumber === undefined) return
     if (isToken(asset.assetId) && !inboundAddressData) return
 
+    await checkLedgerAppOpenIfLedgerConnected(asset.chainId)
+    if (accountNumber === undefined) return
```

Comment:
`accountNumber` check needs to be last, we may not have an accountNumber (which is incorrect as we let users proceed, but another bug)

Reply:
https://github.com/shapeshift/web/issues/7634

Reply:
I don't understand this change. Why would we care about checking if an app is open without the `accountNumber` to be able to continue anyways? Previous diff looks sane to me (only check if app is open just prior to build and broadcast)

Reply:
Just for the sake of completeness! Allows for it to be implemented everywhere for LP, even though the Tx won't go through withuot an accountNumber (as you've noticed, this is reverted in https://github.com/shapeshift/web/pull/7635)